documentation/principles/cost-effective.md

```diff
@@ -2,12 +2,17 @@
 layout: sub-navigation
 order: 105
 title: Cost Effective
-excerpt: Short extract
+excerpt: We aim to help customers reduce build and run costs.
 tags:
 - principle
 eleventyNavigation:
   parent: principles
 ---
-Core Cloud operates consumption-based billing meaning teams only pay for their usage.\
-This incentivises teams to take responsibility and optimise resources attributed to them.\
-Core Cloud supports teams in optimising their resources using techniques like autoscaling, downscaling, spot pricing and ARM powered instances.
\ No newline at end of file
+**We operate consumption-based billing**. \
+Customers are charged on a consumption model and are incentivised to optimise their resources. 
+
+**We support customers in optimising their resources** using techniques like autoscaling, downscaling, spot pricing and ARM powered instances.
```

Comment:
Only a tiny iteration of @nickmcmahon01's suggestion. Changing `will` to `can` i.e. if you get your autoscaling wrong it can increases costs and that responsibility sits with the customer in this context.
```suggestion
**We support customers in optimising their resources.** \
Techniques like autoscaling, downscaling, spot pricing and ARM powered instances can lower costs and improve performance.
```